lib/moirai/engine.rb

```diff
@@ -9,7 +9,8 @@ class Engine < ::Rails::Engine
     end
 
     config.after_initialize do
-      if ActiveRecord::Base.connection.data_source_exists?("moirai_translations")
+      I18n.original_backend = I18n.backend
+      if ActiveRecord::Base.connection.data_source_exists?("moirai_translations") || ENV["RAILS_ENV"] == "test"
```

Comment:
Thanks @coorasse for adding the CI logs! From this and some debugging I noticed that this was false and so the backend was only made up of the old `I18n::Simple` backend - not Moirai. 

I have therefore changed this so that it runs in `test` env and it now works... but it feels a bit hacky. Can you think of any other way? It basically seems that this is being checked **before** the migrations are run but I can't see where I can configure this in the test setup.

```
moirai disabled: tables have not been generated yet.
moirai disabled: tables have not been generated yet.
  [1m[35m (0.2ms)[0m  [1m[35mDROP TABLE IF EXISTS "moirai_translations"[0m
  [1m[35m (0.8ms)[0m  [1m[35mCREATE TABLE "moirai_translations" ("id" integer PRIMARY KEY AUTOINCREMENT NOT NULL, "file_path" varchar, "locale" varchar NOT NULL, "key" varchar NOT NULL, "value" text, "created_at" datetime(6) NOT NULL, "updated_at" datetime(6) NOT NULL)[0m
  [1m[35m (0.1ms)[0m  [1m[35mCREATE TABLE "schema_migrations" ("version" varchar NOT NULL PRIMARY KEY)[0m
  [1m[36mActiveRecord::SchemaMigration Load (0.1ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  [1m[35m (0.1ms)[0m  [1m[32mINSERT INTO "schema_migrations" (version) VALUES (20241028191428)[0m
  [1m[35m (0.2ms)[0m  [1m[32mINSERT INTO "schema_migrations" (version) VALUES
(20241022085549);[0m
  [1m[35m (0.2ms)[0m  [1m[35mCREATE TABLE "ar_internal_metadata" ("key" varchar NOT NULL PRIMARY KEY, "value" varchar, "created_at" datetime(6) NOT NULL, "updated_at" datetime(6) NOT NULL)[0m
  [1m[36mActiveRecord::InternalMetadata Load (1.6ms)[0m  [1m[34mSELECT * FROM "ar_internal_metadata" WHERE "ar_internal_metadata"."key" = ? ORDER BY "ar_internal_metadata"."key" ASC LIMIT 1[0m  [[nil, "environment"]]
  [1m[36mActiveRecord::InternalMetadata Create (0.1ms)[0m  [1m[32mINSERT INTO "ar_internal_metadata" ("key", "value", "created_at", "updated_at") VALUES ('environment', 'test', '2024-11-21 12:40:16.187232', '2024-11-21 12:40:16.187234') RETURNING "key"[0m
  [1m[36mActiveRecord::InternalMetadata Load (0.1ms)[0m  [1m[34mSELECT * FROM "ar_internal_metadata" WHERE "ar_internal_metadata"."key" = ? ORDER BY "ar_internal_metadata"."key" ASC LIMIT 1[0m  [[nil, "environment"]]
  [1m[36mActiveRecord::InternalMetadata Load (0.0ms)[0m  [1m[34mSELECT * FROM "ar_internal_metadata" WHERE "ar_internal_metadata"."key" = ? ORDER BY "ar_internal_metadata"."key" ASC LIMIT 1[0m  [[nil, "schema_sha1"]]
  [1m[36mActiveRecord::InternalMetadata Create (0.1ms)[0m  [1m[32mINSERT INTO "ar_internal_metadata" ("key", "value", "created_at", "updated_at") VALUES ('schema_sha1', '35cf2930fb42e01581f9c280565681eb6759a086', '2024-11-21 12:40:16.188129', '2024-11-21 12:40:16.188131') RETURNING "key"[0m
  [1m[36mActiveRecord::SchemaMigration Load (0.1ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  [1m[36mTRANSACTION (0.0ms)[0m  [1m[36mbegin transaction[0m
 ```